### PR TITLE
remove the block fill rate limit of 70% when farming a block

### PR DESCRIPTION
**Pull request by arvidn**
### Purpose:

to increase transaction rate

### Current Behavior:

Farmers only fill blocks to 70% full

### New Behavior:

Farmers fill blocks all the way. But blocks are compressed and have potential to fit more transactions.

### CODE REVIEW

**Review by AmineKhaldi — DISMISSED** [review]

File: chia/_tests/core/mempool/test_mempool_manager.py [hunk 1]
```diff
@@ -1081,7 +1081,7 @@ async def make_and_send_big_cost_sb(coin: Coin) -> None:
         g1 = sk.get_g1()
         sig = AugSchemeMPL.sign(sk, IDENTITY_PUZZLE_HASH, g1)
         aggsig = G2Element()
-        for _ in range(169):
+        for _ in range(242):
```
Comment (emlowe) [inline chia/_tests/core/mempool/test_mempool_manager.py:1084]: do you want to comment what this magic number is

Comment (arvidn) [reply chia/_tests/core/mempool/test_mempool_manager.py:1084]: yeah, it's not great. it's the number of transactions we need to fill the mempool sufficiently to trigger the expected number of rejections. going from fill rate 70% -> 100%, I simply took 169 / 0.7 to increase this number proportionally. I'll try to improve this a bit.

**Review by emlowe — DISMISSED** [review]

File: chia/_tests/core/mempool/test_mempool_manager.py (hunk 1 above)
Comment (AmineKhaldi) [reply chia/_tests/core/mempool/test_mempool_manager.py]: Sent https://github.com/Chia-Network/chia-blockchain/pull/19026 as an attempt to clarify this.

Comment (emlowe) [reply chia/_tests/core/mempool/test_mempool_manager.py]: I think I might prefer the simpler construct with a comment like Amine has above. The latest version here Arvid has more magic numbers :-)

**Review by emlowe — APPROVED** [review]

File: chia/_tests/core/mempool/test_mempool_manager.py (hunk 1 above)
Comment (arvidn) [reply chia/_tests/core/mempool/test_mempool_manager.py]: I think magic numbers are "more magic" the harder they are to explain. And breaking this down to how the magic number is computed makes the new ones less magical. (and I think in the most recent update there no magic numbers). As long as you accept that the CLVM cost of returning a quoted list is 44 and that it takes 88 bytes to serialize another agg-sig condition.

There are a lot of mempool tests with magic number that need tweaking when tweaking constants. Now we're moving this test to have its expectations properly rooted in the test constants it's using. I think this really helps humans understand (and gain confidence in) the test. I will follow up (in a separate PR) to make sure this test

**Review by AmineKhaldi — APPROVED** [review]

**Pull request merged**